mobile/src/components/category-list/category-list.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
```

Comment:
It's a specific component and we only store screens on a top level, move it to src/components/courses/components/category-list.tsx